lib/Pipfile

```diff
@@ -40,6 +40,7 @@ opencv-python = "*"
 # add to the dev dependencies above, instead.
 
 [packages]
+astor = "*"
```

Comment:
where are we using it?

Reply:
It's being used in https://github.com/streamlit/streamlit/blob/develop/lib/streamlit/caching.py#L115 but we were missing the package in the pipfile and locks